src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.consent.http.service;
+
+import org.broadinstitute.consent.http.db.DataSetDAO;
+
+import javax.inject.Inject;
+import java.util.Date;
+import org.broadinstitute.consent.http.models.DataSet;
+
+
+public class DatasetService {
+
+    private final DataSetDAO dataSetDAO;
+
+    @Inject
+    public DatasetService(DataSetDAO dataSetDAO) {
+        this.dataSetDAO = dataSetDAO;
+    }
+
+    public DataSet createTestDataSet(String json) {
+        return new DataSet(json);
+    }
+
+    public Integer createDataset(String name, String objectId, Boolean active, Integer alias) {
```

Comment:
left this here so that the DatasetService class would use the DataSetDAO it contains (without flagging codacy)

Reply:
👍 For final implementation, we'll return a full dataset. This is good for this PR.